docs/dataframe_api.ipynb

```diff
@@ -0,0 +1,676 @@
+{
```

Comment:
<p>I'd suggest "is not supported by TB's main GUI currently" -&gt; "is not yet supported in TensorBoard's GUI"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/tensorboard/pull/3744/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
Done.